.maintain/monitoring/grafana-dashboards/substrate-networking.json

```diff
@@ -0,0 +1,2728 @@
+{
+  "__inputs": [
+    {
+      "name": "VAR_DATA_SOURCE",
+      "type": "constant",
```

Comment:
Instead of having `constant` inputs, why not have these be selected at the top of the dashboard. Should make it easier for people to test out the dashboard, e.g. via `cd paritytech/substrate && docker-compose -f .maintain/sentry-node/docker-compose.yml up`.

What do you think?

Reply:
I have no idea what is idiomatic, so you (or someone) has to guide me.
I did realize after opening this PR that Grafana's UI didn't like when the data source is a constant, so I assume that this is indeed not the right way to go.